src/schema.ts

```diff
@@ -131,6 +132,14 @@ export interface IWebSettings {
   [key: string]: string | boolean
 }
 
+export interface IHooks {
+  Title?: string
+  Url: string
+  Method: string
+  Headers?: Headers
+  Body?: Body
```

Comment:
This is a default type in TypeScript, right?

Reply:
Yes, same with Headers.